pkg/server/cloud/aws/network.go

```diff
@@ -316,57 +316,97 @@ func (e *AwsEC2) ModifySourceDestinationCheck(instanceID string, isEnabled bool)
 	return err
 }
 
-func (e *AwsEC2) RemoveRoute(destinationCIDR string) error {
+func (e *AwsEC2) RemoveRoute(destinationCIDR, instanceID string) error {
```

Comment:
I like how this function turned out!  It's easy to understand what's going on in multiple cases.